schema/3pse/file/applications/espresso/7.2/pw.x/system.json

```diff
@@ -0,0 +1,638 @@
+{
+    "$id": "3pse/file/applications/espresso/7.2/pw.x/system",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "system schema",
+    "type": "object",
+    "additionalProperties": false,
+    "anyOf": [
+        {
+            "properties": {
+                "celldm": {
+                    "type": "array",
+                    "items": {
+                        "type": "number"
+                    },
+                    "minLength": 6,
+                    "maxLength": 6
+                }
+            }
+        },
+        {
+            "properties": {
+                "A": {
+                    "type": "number"
+                },
+                "B": {
+                    "type": "number"
+                },
+                "C": {
+                    "type": "number"
+                },
+                "cosAB": {
+                    "type": "number"
+                },
+                "cosAC": {
+                    "type": "number"
+                },
+                "cosBC": {
+                    "type": "number"
+                }
+            }
+        }
+    ],
+    "properties": {
+        "ibrav": {
+            "type": "integer"
+        },
+        "nat": {
+            "type": "integer",
+            "description": "number of atoms in the unit cell (ALL atoms, except if space_group is set, in which case, INEQUIVALENT atoms)"
+        },
+        "ntyp": {
+            "type": "integer",
+            "description": "number of types of atoms in the unit cell"
+        },
+        "nbnd": {
+            "type": "integer",
+            "description": "Default: for an insulator, nbnd = number of valence bands (nbnd = # of electrons /2); for a metal, 20% more (minimum 4 more)"
+        },
+        "nbnd_cond": {
+            "type": "integer",
+            "description": "Default: nbnd_cond = nbnd - # of electrons / 2 in the collinear case; nbnd_cond = nbnd - # of electrons in the noncollinear case."
+        },
+        "tot_charge": {
+            "type": "number",
+            "default": 0
+        },
+        "starting_charge": {
+            "type": "number",
+            "default": 0,
+            "description": "starting charge on atomic type 'i', to create starting potential with startingpot = 'atomic'."
+        },
+        "tot_magnetization": {
+            "type": "number",
+            "default": -10000,
+            "description": "Total majority spin charge - minority spin charge. Used to impose a specific total electronic magnetization. If unspecified then tot_magnetization variable is ignored and the amount of electronic magnetization is determined during the self-consistent cycle."
+        },
+        "starting_magnetization": {
```

Comment:
`starting_magnetization` can have values in the range [-1, 1]. We may specify `minimum` and `maximum`.